plugins/SSH.koplugin/main.lua

```diff
@@ -29,7 +32,7 @@ function SSH:start()
         "./dropbearmulti dropbear",
         "-E", "-r settings/SSH/dropbear_rsa_host_key",
```

Comment:
Same here as the remark I just posted below at the InfoMessage. Should use something like `DataStorage:getSettingsDir()..SSH/dropbear_rsa_host_key`

Reply:
Hum, that's more tricky, because the keyfile is hardcoded to settings/SSH/authorized_keys, so if  DataStorage:getSettingsDir change, it can ends up weird. I prefer to just use DataStorage:getFullDataDir directly.

Reply:
Weird how? Hardcoded where?

Reply:
It's coming from here : https://github.com/koreader/koreader-base/pull/640/files#diff-9d90386d5bae2bf2c8181b518acc30cfR9


So to sum up, in the settings/SSH/ directory, there should be two files : 

 - dropbear_rsa_host_key which is use in the command line
 - authorized_keys which is hardcoded in dropbear via the linked patch

So if DataStorage:getSettingsDir() is different than DataStorage:getSettingsDir()/settings, then dropbear_rsa_host_key will follow, but not authorized_keys.

So I choosed to only use DataStorage:getSettingsDir()/settings and be done with it.
I don't have a better idea ( kinda don't want to make another patch to dropbear ), so I'm open to suggestions.

Reply:
Since this isn't meant to run on the platforms where it matters we can let it pass (with a warning comment please), but I think the salient point is being overlooked here. `DataStorage:getFullDataDir()` can be anything in actual practice right now. It's not the same thing as the directory with the binaries!

`DataStorage:getSettingsDir()` could theoretically be something different than `DataStorage:getFullDataDir().."/settings"` but that's much less likely to actually happen. It's more of an architecturally sound convenience function.